packages/xsnap/example/modules/after.js

```diff
@@ -0,0 +1,2 @@
+import increment from "./increment.js";
```

Comment:
and maybe delete the whole `example/` directory, if it's mostly about demonstrating that snapshots work correctly..

Reply:
Ideally we'd turn them into unit tests...

But I agree that I don't really want to maintain them here.

Reply:
I’m removed the examples. We can put them back if we also write automated verification for them.